schedule/schedule-appengine/src/main/i18n/locale-en.json

```diff
@@ -87,7 +85,6 @@
   "ENTER_ABOUT_ME": "Enter something about me",
   "ENTER_ACTIVITY_END_DATE": "Enter Activity end date",
   "ENTER_CAPACITY": "Enter capacity",
-  "ENTER_CURRENT_PASSWORD": "Enter current password",
```

Comment:
What we were discussion earlier about translations.  It is very difficult to know if this key `ENTER_CURRENT_PASSWORD` is user in another template or not...